packages/anthropic/src/anthropic-messages-language-model.ts

```diff
@@ -966,8 +982,23 @@ export class AnthropicMessagesLanguageModel implements LanguageModelV2 {
                 usage.cachedInputTokens =
                   value.message.usage.cache_read_input_tokens ?? undefined;
 
+                const rawProviderUsage =
+                  typeof chunk.rawValue === 'object' &&
+                  chunk.rawValue !== null &&
```

Comment:
also a null check first makes more sense imo, because it would exit early

Reply:
i wonder why we need those typechecks. since this is coming from anthropic, would it be cleaner to extend the zod schema for parsing?